lang/po/Firmware_nl.po

```diff
@@ -0,0 +1,1787 @@
+msgstr "Opwarmen invoeren"
+
+# 
+#: ultralcd.cpp:2456
+msgid "Preheating to unload"
+msgstr "Opwarmen uitwerpen"
+
+# MSG_SELFTEST_PRINT_FAN_SPEED c=18
+#: ultralcd.cpp:8414
+msgid "Print fan:"
+msgstr "Print fan:"
+
+# MSG_CARD_MENU
+#: messages.c:21
+msgid "Print from SD"
+msgstr "Print van SD"
+
+# 
+#: ultralcd.cpp:2301
+msgid "Press the knob"
+msgstr "Druk op knop"
+
+# MSG_PRINT_PAUSED c=20 r=1
+#: ultralcd.cpp:1074
+msgid "Print paused"
+msgstr "Print pauzeren"
+
+# 
+#: mmu.cpp:723
+msgid "Press the knob to resume nozzle temperature."
+msgstr "Druk op de knop om de temperatuur van de tuit te hervatten."
+
+# MSG_FOLLOW_CALIBRATION_FLOW c=20 r=8
+#: messages.c:41
+msgid "Printer has not been calibrated yet. Please follow the manual, chapter First steps, section Calibration flow."
+msgstr "Printer is nog niet gekalibreerd. Volg de handleiding, hoofdstuk First steps, sectie Calibration flow."
+
+# 
+#: ultralcd.cpp:1707
+msgid "Print FAN"
+msgstr "Print fan"
+
+# MSG_PRUSA3D
+#: ultralcd.cpp:2189
+msgid "prusa3d.com"
+msgstr ""
+
+# MSG_BED_CORRECTION_REAR c=14 r=1
+#: ultralcd.cpp:3285
+msgid "Rear side [um]"
+msgstr "Achterkant[um]"
+
+# MSG_RECOVERING_PRINT c=20 r=1
+#: Marlin_main.cpp:9780
+msgid "Recovering print    "
+msgstr "Print herstellen    "
+
+# MSG_REMOVE_OLD_FILAMENT c=20 r=4
+#: mmu.cpp:830
+msgid "Remove old filament and press the knob to start loading new filament."
+msgstr "Verwijder de oude filament en druk op de knop om nieuwe filament te laden."
+
+# 
+#: 
+msgid "Prusa i3 MK3S OK."
+msgstr ""
+
+# MSG_CALIBRATE_BED_RESET
+#: ultralcd.cpp:5844
+msgid "Reset XYZ calibr."
+msgstr "Reset XYZ kalibr."
+
+# MSG_BED_CORRECTION_RESET
+#: ultralcd.cpp:3286
+msgid "Reset"
+msgstr ""
+
+# MSG_RESUME_PRINT
+#: ultralcd.cpp:6831
+msgid "Resume print"
+msgstr "Print hervatten"
+
+# MSG_RESUMING_PRINT c=20 r=1
+#: messages.c:73
+msgid "Resuming print"
+msgstr "Hervatten print"
+
+# MSG_BED_CORRECTION_RIGHT c=14 r=1
+#: ultralcd.cpp:3283
+msgid "Right side[um]"
+msgstr "Recht.kant[um]"
+
+# MSG_SECOND_SERIAL_ON c=17 r=1
+#: ultralcd.cpp:5759
+msgid "RPi port     [on]"
+msgstr "RPi port    [aan]"
+
+# MSG_SECOND_SERIAL_OFF c=17 r=1
+#: ultralcd.cpp:5757
+msgid "RPi port    [off]"
+msgstr "RPi port    [uit]"
+
+# MSG_WIZARD_RERUN c=20 r=7
+#: ultralcd.cpp:4877
+msgid "Running Wizard will delete current calibration results and start from the beginning. Continue?"
+msgstr "Starten van de Wizard verwijdert de huidige kalibreringsresultaten en begint vanaf het begin. Doorgaan?"
+
+# MSG_TOSHIBA_FLASH_AIR_COMPATIBILITY_OFF c=19 r=1
+#: ultralcd.cpp:5389
+msgid "SD card  [normal]"
+msgstr "SD card [normaal]"
+
+# MSG_TOSHIBA_FLASH_AIR_COMPATIBILITY_ON c=19 r=1
+#: ultralcd.cpp:5387
+msgid "SD card [flshAir]"
+msgstr "SD Karte[flshAir]"
+
+# 
+#: ultralcd.cpp:3012
+msgid "Right"
+msgstr "Rechts"
+
+# MSG_FIND_BED_OFFSET_AND_SKEW_LINE1 c=60
+#: messages.c:38
+msgid "Searching bed calibration point"
+msgstr "Zoeken bed kalibratiepunt"
```

Comment:
would suggest "Bed kalibratiepunt aan het zoeken" if there is sufficient space.

Reply:
Bed isn't an adjective here, so it can be Bedkalibratiepunt I guess